http/logical.go

```diff
@@ -75,18 +77,29 @@ func buildLogicalRequest(core *vault.Core, w http.ResponseWriter, r *http.Reques
 				data = getData
 			}
 		}
+		if path == "sys/storage/raft/snapshot" {
```

Comment:
Do we also have to check for `path == "sys/storage/raft/snapshot-force"`?

Reply:
Yeah I think so too

Reply:
No, this is the `GET` case which is only supported by `snapshot`